testsuite/tests/lib-systhreads/boundscheck.ml

```diff
@@ -0,0 +1,39 @@
+(* TEST
+
+include systhreads
+* hassysthreads
+** not-windows
```

Comment:
How come?

Reply:
If testing non-systhreads and/or windows is desired, it looks like this bug can be triggered using multiple domains (see #11425).